src/drivers/device/nuttx/I2C.cpp

```diff
@@ -235,38 +216,44 @@ I2C::transfer(const uint8_t *send, unsigned send_len, uint8_t *recv, unsigned re
 	} while (retry_count++ < _retries);
 
 	return ret;
+}
 
+int
+I2C::read(unsigned address, void *data, unsigned count)
+{
+	uint8_t cmd = address;
+	return transfer(&cmd, 1, (uint8_t *)data, count);
 }
 
 int
-I2C::transfer(i2c_msg_s *msgv, unsigned msgs)
+I2C::write(unsigned address, void *data, unsigned count)
```

Comment:
Is this an API change? Or just a diff artifact?

Reply:
I dropped the `I2C::transfer(i2c_msg_s *msgv, unsigned msgs)` because it wasn't used. The regular transfer is still there.

Reply:
I think that is reasonable. If we feel we need it in the future we can bring it back.